src/pages/unlock.tsx

```diff
@@ -48,11 +60,20 @@ export function Unlock(): JSX.Element {
       setError('The password you entered is invalid.');
     }
     setLoading(false);
+    stopWaitingMessage();
     const unlockSuccessTimeMs = performance.now();
     void analytics.track('complete_unlock', {
       durationMs: unlockSuccessTimeMs - startUnlockTimeMs,
     });
-  }, [analytics, unlockWallet, password, decodedAuthRequest, navigate]);
+  }, [
+    analytics,
+    startWaitingMessage,
```

Comment:
If I'm not mistaken, this will be redeclared on every render, as the reference to the function is new (on every render).